resolwe/flow/management/commands/runexecutor_listener.py

```diff
@@ -0,0 +1,28 @@
+""".. Ignore pydocstyle D400.
+
+=================================
+Standalone Executor Contact Point
+=================================
+
+Command to run on local machine::
+
+    ./tests/manage.py runexecutor_listener
+
+"""
+
+from django.core.management.base import BaseCommand
+
+from resolwe.flow.managers.listener import ExecutorListener
+
+
+class Command(BaseCommand):
+    """Run the executor listener."""
+
+    help = 'Run the standalone manager contact point for executors.'
+
+    def handle(self, *args, **kwargs):
+        """Run the executor listener. This method never returns."""
+        listener = ExecutorListener()
+        listener.start()
+        # listener.terminate() left out on purpose, this method never returns
+        listener.join()
```

Comment:
Can this be changed to handle reload-on-update (only in development mode) or would this complicate things too much?

Reply:
You mean reloading when the underlying source files change? This would be difficult, since it's a manually spawned process, meaning that any file change monitoring would also have to be manual, I think.

Reply:
Check how it is done in observers: https://github.com/genialis/django-rest-framework-reactive/commit/1b253ebcd91fb4412f474a553e7a91b0e72cf672